### DISCUSSION TITLE
Custom Injector

### DISCUSSION THREAD

**Discussion by Jonny11bonk** [danielkrupinski/Osiris#3065] [Ideas]
I would like to see Osiris to have Injector already built-in and saving the configs and other files in into the same folder but not in the Documents. It would be much safer to isolate the Osiris files from the Windows, for example running Osiris from a removable drive where the configs would make.

**Comment by stayindead** [comment]
no,

**Comment by gra439** [comment]
Well... a little bit of coding, and one compile and you can do the saving the configs/other files. Usually git will prevent your "pulls" (if ya know GitHub) from overwriting your code, and as long as there aren't MAJOR updates to the way Osiris saves/loads configs, you don't ever have to change it.

As for Injectors, it's probably NOT a good idea. The number of people that use this cheat, and would then therefore use the built in injector (if it was made) would probably make it very detectable, and the fact that valve would also be able to see the source code to the injector, and could then simply compile and get the "signature" for it would make it so very detected.

**Reply by PEXCH-SENTXMENT** [reply]
> Well... a little bit of coding, and one compile and you can do the saving the configs/other files. Usually git will prevent your "pulls" (if ya know GitHub) from overwriting your code, and as long as there aren't MAJOR updates to the way Osiris saves/loads configs, you don't ever have to change it.
> 
> As for Injectors, it's probably NOT a good idea. The number of people that use this cheat, and would then therefore use the built in injector (if it was made) would probably make it very detectable, and the fact that valve would also be able to see the source code to the injector, and could then simply compile and get the "signature" for it would make it so very detected.

+
